lib/src/analytics.dart

```diff
@@ -41,8 +51,9 @@ abstract class Analytics {
     required int toolsMessageVersion,
     required String toolsMessage,
     required FileSystem fs,
```

Comment:
By making them optional, tests that need to verify the file system can provide it, and the others don't need to provide it.

Reply:
Makes sense, I've made them optional with default values for the message and version number, and if the filesystem isn't specified in the constructor, it will fall back on a default instance `MemoryFileSystem.test(style: ...)`